pennylane/measurements/mid_measure.py

```diff
@@ -243,7 +243,7 @@ def _create_mid_measure_primitive():
         measurement.
 
     """
-    # pylint: disable=import-outside-toplevel
+    # pylint: disable=import-outside-toplevel, no-member
```

Comment:
What was having `no-member`?

Reply:
Can we localize the silencing to that line?

Reply:
I generally prefer more specific pylint-silencing in source code, as it avoids false negatives. Though I guess `no-member` is a fairly harmless one.

Reply:
I'll localize it. Thanks for the suggestion :)